src/app/components/AmpExperiment/index.tsx

```diff
@@ -32,17 +38,31 @@ const AmpHead = () => (
   </Helmet>
 );
 
-const AmpExperiment = ({ experimentConfig }: AmpExperimentProps) => {
+const AmpScript = ({ config }: { config: Record<string, unknown> }) => {
```

Comment:
**Suggestion (non-blocking)** For readability/simplicity, I would pull this out into a AmpScriptProps type. If the codebase convention is to do this inline then happy to continue with that as I don't know it well enough.